examples/error.rs

```diff
@@ -36,7 +36,7 @@ fn main() {
                 &std::collections::HashMap::from([
                     ("Mercury".to_owned(), "tiny".to_owned()),
                     ("Venus".to_owned(), "hot".to_owned()),
-                    ("Earth".to_owned(), "just right".to_owned()),
+                    ("Earth".to_owned(), "just right2".to_owned()),
```

Comment:
this copied data structure was making complaints in my ide for duplicated code, but this data structure would be annoying to import, i thought. other opinions welcome.